examples/adb_cli.rs

```diff
@@ -34,6 +34,8 @@ pub enum Command {
     TrackDevices,
     /// Lists available server features.
     HostFeatures,
+    /// Pushes 'filename' to the 'path' on device
+    Push { filename: String, path: String },
```

Comment:
Can you please remove this subcommand from here and add it in a new PR ? ;)

Reply:
I have no idea how these got submitted with the host feature patch. This was something I was testing a long time ago and was going to clean up, but then I ran into to host-feature thing. 

I'll clean up the patch and resubmit.

Reply:
Ah, now I understand what happened. It's me not understanding how PRs work on github. I'll fix.